pkg/pod/admission/nodeenv/admission.go

```diff
@@ -0,0 +1,97 @@
+package admission
```

Comment:
I think I would place this file under /pkg/project/admission/nodeenv/admission.go given that the nodeSelector is a feature of the project.